File: Linked List/2. Doubly Linked List/1. Doubly LinkedList Creation/DoublyLinkedList.java

class DLLNode
{
    int value;
    DLLNode prev;
    DLLNode next;
    
    DLLNode()
    {
        this.prev = null;
        this.next = null;
    }

    DLLNode(int value)
    {
        this.value = value;
        this.prev = prev;
        this.next = next;
    }
}

public class DoublyLinkedList
{
    private DLLNode head;

    public DoublyLinkedList()
    {
        head = null;
    }

    public boolean isEmpty()
    {
        if (head == null)
            return true;
        else
            return false;
    }

    public static void main(String args[])
    {
        DoublyLinkedList list = new DoublyLinkedList();
        System.out.println(list.isEmpty());
    }
}

File: Linked List/2. Doubly Linked List/1. Doubly LinkedList Creation/DoublyLinkedList.py
class Node:
    def __init__(self, data):
        self.data = data
        self.prev = None
        self.next = None

    def get_data(self):
        return self.data


class DoublyLinkedList:
    def __init__(self):
        self.head = None
    
    def isEmpty(self):
        if self.head == None:
            return True
        else:
            return False

def main():
    dll = DoublyLinkedList();
    print(dll.isEmpty())

if __name__ == "__main__":
    main()